### DISCUSSION TITLE
Problema en visualización de error en campo "Phone"

### DISCUSSION THREAD

**Discussion by VirBruno** [frlp-utn-ingsoft/catedra#146] [Q&A] [answered]
Buenas noches, 

Agregué la validación para que revise si el teléfono es numérico en models.py, al correrlo, el error aparece luego de borrar la información de prueba "incorrecta". No modifiqué el tipo de dato, dado que cuando lo hice (puse integer) al cargarle letras se reinicia el formulario,

MODELS
        elif key =='phone':
            phone_error = validate_phone(field_value)
            if phone_error:
                errors["phone"] = phone_error
                
def validate_phone(number):
    print(number)
    if not(number.isnumeric()):
        print(number)
        return "El teléfono indicado debe contener sólo números"
    return None

![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/e4e9e8ba-0d6d-42b7-8c5d-ca6c844f3be1)
![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/158d5152-2e01-4530-8c82-b688616a6178)

**Comment by VirBruno** [comment]
Caso OK
![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/86481814-0cd2-4a9f-b267-988d80dbb640)

**Comment by VirBruno** [comment]
Caso NO OK
![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/fb4ce9a2-40d7-420f-9397-81da34e242e8)

![imagen](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/7b3377a3-60c9-4b85-98e0-ec9d056c6450)

![Uploading imagen.png…]()

**Comment by RodrigoJacznik** [comment, marked as answer]
Hola Virginia,

Proba poniendo el attributo `type="tel"` y `pattern="\d+"` en el input del telefono
